plaso/analysis/browser_search.py

```diff
@@ -39,6 +38,37 @@ class BrowserSearchPlugin(interface.AnalysisPlugin):
   _EVENT_TAG_COMMENT = 'Browser Search'
   _EVENT_TAG_LABELS = ['browser_search']
 
+  _SUPPORTED_EVENT_DATA_TYPES = frozenset([
```

Comment:
I don't think this is necessary any more, we can just check for a url attribute. I know this was check for WEBHIST before, but this doesn't add much.

Reply:
Let's make these changes separate from this PR. Maybe first do the event data attribute standarization.

https://github.com/log2timeline/plaso/issues/2530